### ISSUE TITLE
Write a Notify Privacy Analysis (PIA-lite)

### ISSUE THREAD

**Issue by sharlychan-cds** [cds-snc/notification-planning#268]
_Give your user story a simple, clear title_

# Description

We have a draft Privacy Impact Assessment (PIA) for Notify. Work was started with TBS ATIP and we need to incorporate their comments, update and re-evaluate it since it was written. A formal PIA is not needed and we're working with OCIO PDPD on a Notify Privacy Analysis document (a PIA-lite).

The goal is to create a technical Notify Privacy Analysis with a privacy compliance-based govt audience. Will re-evaluate to see if we can adapt it to make it public and plain language.

# Acceptance Criteria** (Definition of done) 
- [x] Notify and OCIO exchange information
- [x] OCIO starts the first draft
- [x] Notify reviews and adds to comments
- [x] OCIO to review
- [x] Delivery policy review
- [x] Review by Notify PM
- [x] Review by devs/ SRE (flag specific questions)
- [x] Submit revisions to OCIO for review
- [x] Incorporate changes (if necessary)
- [x] Translation of Privacy Analysis
- [x] Communicate internally to CDS (STT?)
- [x] Evaluate whether content design should be done to make a public version
- [x] Consider if we should mention we have one in our Privacy Statement

**Comment by yaelberger-commits** [comment]
OCIO PDPD is working on this with our policy team

**Comment by amazingphilippe** [comment]
It changed, so Sharly needs to work on it

**Comment by sharlychan-cds** [comment]
Working on comments to return back to OCIO PDPD

**Comment by yaelberger-commits** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Notify-Planning-614b3ad91bc2030015ed22f5/issues/cds-snc/notification-planning/268?planning-poker) with Zenhub @sharlychan-cds

**Comment by yaelberger-commits** [comment]
Sharly will meet with Kayla soon to accept comments

**Comment by YedidaZalik** [comment]
@sharlychan-cds working on and will chat OCIO in next day or so.

**Comment by YedidaZalik** [comment]
Sharly working on and in contact with OCIO

**Comment by YedidaZalik** [comment]
Meeting OCIO today

**Comment by YedidaZalik** [comment]
Met with OCIO and some changes accepted, now working on draft and will ask some questions to Yael and Josh early in week

**Comment by YedidaZalik** [comment]
Sharly is at a meeting with OCIO this morning :)

**Comment by adriannelee** [comment]
Continued worked on this on Friday. Will be ready for review soon

**Comment by andrewleith** [comment]
- @sharlychan-cds to review comments this week

**Comment by YedidaZalik** [comment]
Moving to Sprint backlog until @sharlychan-cds returns
Did some pairing before holidays with table of data we collect, show how collected, stored, use which is one of last pieces of work required.

**Comment by sharlychan-cds** [comment]
February 16, 2023: Sharly and Nisa (Policy) reviewed and sent back final comments to TBS. Waiting for their final approval. Next steps: Giving PM and Head of Notify a tl;dr review of the the document (and any edits needed) before accepting.

**Comment by sharlychan-cds** [comment]
Have a meeting with Josh, Yael to brief them today!

**Comment by sharlychan-cds** [comment]
Josh and Yael have accepted. Now to dev review for specific sections!

**Comment by sharlychan-cds** [comment]
Dev review https://docs.google.com/document/d/1oODAdd-O9j1JlzxzdVHiwXk3zRnNZj__/edit# for pages 5,7-11, 18-19

**Comment by sharlychan-cds** [comment]
@jimleroyer to review to see if the technical descriptions are still accurate https://docs.google.com/document/d/1oODAdd-O9j1JlzxzdVHiwXk3zRnNZj__/edit# for pages 5,7-11, 18-19

Does a deadline of Thursday March 16 work? We're hoping to move this along quickly 🙏

**Comment by sastels** [comment]
waiting for @jimleroyer  / devs to review.

**Comment by jimleroyer** [comment]
@mtoutloff @sharlychan-cds I reviewed and that looks great overall. I left [a few comments](https://docs.google.com/document/d/1oODAdd-O9j1JlzxzdVHiwXk3zRnNZj__/edit#). Please ping me if you have any question!

**Comment by mtoutloff** [comment]
Thanks @jimleroyer! Will review today and let you know if there are any questions.

**Comment by yaelberger-commits** [comment]
Melissa still reviewing remaining risks and syncing with Forms team

**Comment by yaelberger-commits** [comment]
Still with Jimmy for some final reviews

**Comment by yaelberger-commits** [comment]
Melissa has meeting with Jimmy this aft for remaining risk

**Comment by yaelberger-commits** [comment]
Melissa and Jimmy met about auditing
Met with TBS about risk
Melissa documented some key points, will review with Nisa
Dependant on ISA signatures issue (PRDD says about Forms)

**Comment by YedidaZalik** [comment]
Melissa at monthly meeting with Privacy folks today so maybe waiting for info from PRDD or Forms team.

**Comment by mtoutloff** [comment]
Met with PRDD this morning and they are expecting to provide us their advice on the SIN collection for Forms this week. Once we receive it, I will review to see if it has any impacts on the risks identified in the analysis.

**Comment by YedidaZalik** [comment]
No update since last week.

**Comment by YedidaZalik** [comment]
No updates
Still waiting on PRDD but majority of their team is out so moving to product backlog

**Comment by mtoutloff** [comment]
Emailed PRDD about their thoughts on the risks and to get their feedback. Received their response yesterday (June 20) and need to review

**Comment by mtoutloff** [comment]
Emailed final draft analysis to PRDD on June 27th, once reviewed it will be ready for approval.

**Comment by adriannelee** [comment]
Heard back from PRDD yesterday - they're good with our changes but taking through their management (approvals). They'll follow up on approval timelines shortly.

**Comment by mtoutloff** [comment]
Will be following up with PRDD this week for update on the status of their review

**Comment by mtoutloff** [comment]
PRDD review should be completed this week, next step will be to proceed to sign-off

**Comment by mtoutloff** [comment]
Received minor comments from PRDD this week and made a few edits to address their comments. With Nisa for review.

**Comment by mtoutloff** [comment]
Sent back Analysis to PRDD for final review

**Comment by yaelberger-commits** [comment]
Melissa working on a briefing for Leanne and Josh. PRDD gave the thumbs up

**Comment by mtoutloff** [comment]
Draft briefing shared with Mohamed and Yael. Meeting set up to brief Leanne and Josh on Aug 9

**Comment by mtoutloff** [comment]
Decided to finalize the Privacy Analysis with what we know so far following our transition to ESDC and can update it down the road if needed

**Comment by mtoutloff** [comment]
With Nisa for review. Meeting to brief Ioana set up for tomorrow

**Comment by mtoutloff** [comment]
Briefed Leanne: She agrees that we can consider this version of the Privacy Analysis done and dusted: https://docs.google.com/document/d/1L3eLNGW7dJ776XADA7PvEIEbbOx5Uh7BRMicH7KRt4Y/edit#heading=h.1chp7uuqxp8x

**Comment by mtoutloff** [comment]
Sending to translation

**Comment by yaelberger-commits** [comment]
Confirm with Melissa next week if this has been sent to translation

**Comment by mtoutloff** [comment]
With translation

**Comment by mtoutloff** [comment]
Followed up with translation. The doc still needs to be proofread, but there have been many urgent requests, so it's been on hold for now

**Comment by mtoutloff** [comment]
Followed up with translation and they expect have it finished tomorrow (july 23)

**Comment by mtoutloff** [comment]
Followed up with translation and they expect have it finished tomorrow (july 23)

**Comment by whabanks** [comment]
Elise shared French translations.

**Comment by mtoutloff** [comment]
French version was converted to pdf for sharing: https://drive.google.com/drive/folders/1ImDqNMGY1YW3EAyLf73yfPGFvhkaXxT9
